RNTester/js/RNTesterList.macos.js

```diff
@@ -33,11 +33,17 @@ const ComponentExamples: Array<RNTesterExample> = [
     module: require('./DarkModeExample'),
     supportsTVOS: false,
   },
+  {
+    key: 'DatePickerIOSExample',
+    module: require('./DatePickerIOSExample'),
+    supportsTVOS: false,
+  },
+  // [TODO(OSS Candidate ISS#2710739)
   {
     key: 'DatePickerMacOSExample',
```

Comment:
DatePickerMacOSExample is an old copy of DatePickerIOSExample.   Its original intention was just to add a couple test cases for new macOS specific props.    In a later PR I will just add those test cases to DatePickerIOSExample in Platform.OS === 'macos' conditions and then eliminate DatePickerMacOSExample.

At that time, `RNTesterList.macos.js` can just import `RNTesterList.ios.js`